components/cms-services/pkg/runtime/service/service_test.go

```diff
@@ -45,15 +46,30 @@ func TestService_setupHandlers(t *testing.T) {
 				srv.Register(endpoint)
 			}
 			recorder := httptest.NewRecorder()
+
+			metRecorder := httptest.NewRecorder()
 			mux := srv.SetupHandlers()
 			request := httptest.NewRequest(testCase.targetMethod, testCase.targetEndpoint, nil)
+			metricsReq := httptest.NewRequest(http.MethodGet, "/metrics", nil)
 
 			// when
 			mux.ServeHTTP(recorder, request)
 
 			// then
 			g.Expect(recorder.Result().StatusCode).To(gomega.Equal(testCase.expectedStatus))
 
+			// when
+			mux.ServeHTTP(metRecorder, metricsReq)
+
+			// then
+			resp := metRecorder.Result()
+			defer func() {
+				err := resp.Body.Close()
+				g.Expect(err).NotTo(gomega.HaveOccurred())
+			}()
+			body, _ := ioutil.ReadAll(resp.Body)
+			g.Expect(resp.StatusCode).To(gomega.Equal(http.StatusOK))
+			g.Expect(body).To(gomega.Not(gomega.HaveLen(0)))
```

Comment:
Please register custom endpoint, only for checking that is everything is ok, like in other services.

Reply:
Registering custom endpoint is main focus of this test, I've not altered this behaviour